src/CodeGen_LLVM.h

```diff
@@ -593,9 +652,9 @@ class CodeGen_LLVM : public IRVisitor {
     llvm::Function *add_argv_wrapper(llvm::Function *fn, const std::string &name,
                                      bool result_in_argv, std::vector<bool> &arg_is_buffer);
 
-    llvm::Value *codegen_dense_vector_load(const Type &type, const std::string &name, const Expr &base,
-                                           const Buffer<> &image, const Parameter &param, const ModulusRemainder &alignment,
-                                           llvm::Value *vpred = nullptr, bool slice_to_native = true);
+    llvm::Value *codegen_vector_load(const Type &type, const std::string &name, const Expr &base,
```

Comment:
At some point in the future we need to try to clean up this and related APIs; IMHO having 6-9 args (w/ the last three having defaults) makes for unreasonably hard to read code at the call site (but that is of course irrelevant for this PR)